frontend/src/components/CustomTable.tsx

```diff
@@ -406,6 +458,35 @@ export default class CustomTable extends React.Component<CustomTableProps, Custo
     return result;
   }
 
+  public handleChange = (name: string) => (event: any) => {
+    const value = event.target.value;
+    this.setStateSafe({ [name]: value } as any,
+      async () => {
+        if (name === 'filterString') {
```

Comment:
If we rename the function, maybe remove this line?

Reply:
Nope. Removed